assignment02/js/script.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready(function(){
+var femalename1;
+femalename1= "Haley"
+var femalename1 = prompt("Please enter a female name (1 of 5)");
+//(femalename1);
+$('#female-name1').text(femalename1);
+state1 = "New York"
+var state1 = prompt("Please enter a state name");
+state1 = "New York"
+var geographical;
+geographical = "mountain"
+var geographical = prompt("Please enter a geographical loction (1 of 2)");
+var malename;
+malename = "Nate"
+var malename = prompt("Please enter a male name (1 of 2)");
+var occupation;
+occupation = "teacher"
+var occupation = prompt("Please enter an occupation");
+var femalename2;
+femalename2 = "Jenna"
+//(femalename1);
+$('#female-name2').text(femalename);
+var noun1;
+noun1 = "paper"
+var noun1 = prompt("Please enter a noun (1 of 5),");
+var noun2;
+//(noun1);
+$('#noun2').text(noun2);
+noun2 = "pen"
+var malename2;
+malename2 = "Ryan"
+var femalename3;
+femalename3 = "Lauren"
```

Comment:
Here you are assigning values without giving the reader an opportunity. I suspect you did this because the assignment says not to prompt more than 12 times. This is because you only have 12 unique variables you need to complete the paragraph - though some of those variables will be assigned to multiple IDs. Does this help? Please reach out if not.